### DISCUSSION TITLE
Unable to play DRM protected content

### DISCUSSION THREAD

**Discussion by sathhyanbs** [cypress-io/cypress#26068] [Questions and Help]
I am Facing an issue while trying to play a DRM protected content. Able to play Trailers (NON-DRM). But while playing a movie (DRM protected) observing a error pop as "you are in your private incognito mode."

Any help with how to fix this or an example would help a lot.
![Screenshot (2)](https://user-images.githubusercontent.com/119301867/224090629-bd2df9b8-be93-4a50-9383-a89d8156ff5f.png)

**Comment by fallenreaper** [comment]
As far as I am tracking, Cypress enables the watching of non-DRM footage, which is why you have been successful in viewing the ads, however the underlying code does not allow Cypress to stream DRM content.  If you have non-DRM video content, you would easily see that it can be viewed. 

I was looking into something similar myself, and was thinking after a few years cypress would enable such a thing.  They still do not.  From more than a few blogs, it seems more likely that the underlying browsers have a lot of plugins disabled for security reasons.  I was looking at the chrome components to compare and contrast, and it has none for cypress, where normal browsers, you will see widevine which is the DRM Decoder.
